magento/magento2
# prevent deletion of system attributes (through the REST API)

### Description (*)
REST API attribute delete lets you delete system attributes

### Fixed Issues (if relevant)

1. magento/magento2#23453: REST API attribute delete lets you delete system attributes

### Manual testing scenarios (*)

1. Ran URL in postman: rest/all/V1/products/attributes/77
2. Request Type: Delete
3. Before Fix: Attribute was deleted irrelevant of system or user defined
4. After Fix: Message is displayed if the attribute requested to be deleted is system attribute (i.e Not User Defined)

### Contribution checklist (*)
